checkDetail.go

```diff
@@ -244,6 +244,12 @@ func (cd *CheckDetail) String() string {
 // Validate performs imagecashletter format rule checks on the record and returns an error if not Validated
 // The first error encountered is returned and stops the parsing.
 func (cd *CheckDetail) Validate() error {
+	return cd.ValidateWithOpts(ValidateOpts{}) // all options should be defined so the default value (false) results in the default (current) behavior
+}
+
+// ValidateWithOpts performs imagecashletter format rule checks with validate opts
+func (cd *CheckDetail) ValidateWithOpts(opts ValidateOpts) error {
```

Comment:
I don't think each record needs a separate `ValidateWithOpts(..)` func. Also the default behavior should assume that the default `ValidateOpts` are specified, but just check for nil opts. 

https://github.com/moov-io/ach/blob/v1.31.3/batch.go#L323-L328